portfolio/src/main/java/com/google/sps/servlets/newServlet.java

```diff
@@ -0,0 +1,19 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that returns HTML that contains the page view count. */
+@WebServlet("/mypage.html")
+public class newServlet extends HttpServlet {
+
+  private int pageViews = 0;
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    response.getWriter().println("<h1>Hello world!</h1>");
```

Comment:
technically the code and the docs from line 9 don't match. You aren't returning the page views, yet.